assets/html/custombar.html

```diff
@@ -7,18 +21,33 @@
       <div class="bounce3"></div>
     </div>
   </div>
+  <label class="letter-label">
+        <input class="js-bag-input letters-input" type="text" maxLength="2" pattern="[A-Z]" placeholder="A-Z"/>
+  </label>
+  <div class="tab-cnr">
+    <div class="tab-buffer">
```

Comment:
the tab buffer is for the bottom border so that the active tab does not have a border...is there a better way to do that in the css? like specifying how far the border should go?